libraries/chain/webassembly/console.cpp

```diff
@@ -62,39 +62,39 @@ namespace eosio { namespace chain { namespace webassembly {
       });
    }
 
-   void interface::printui128(const unsigned __int128& val) {
+   void interface::printui128(legacy_ptr<const unsigned __int128> val) {
 		predicated_print(context,
       [&]() {
 			fc::uint128_t v(val>>64, static_cast<uint64_t>(val) );
```

Comment:
Again, this is concerning. What is actually being cast into a `uint64_t` here after the change to the function signature? If we just treat `val` as a pointer (rather maybe a pointer or maybe a reference), it would be very clear that we must write `fc::uint128_t v(*val >> 64, static_cast<uint64_t>(*val) );` here.